src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -1,61 +1,69 @@
 
-/**
- * @typedef {Object} ProfilePicturePropsParam
- * @property {function} onImageContainerClick
- * @property {String} profilePicPath
- * @property {Boolean} disabledAddIcon
- */
-/**
- *
- * @param {ProfilePicturePropsParam} props
- */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
+  return (
+    <View style={styles.wrapImageProfile}>
+      <TouchableNativeFeedback onPress={onImageContainerClick}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2)}>
+          <View style={styles.profileImageContainer}>
             <Image
-                style={styles.profileImage}
-                source={{
-                uri: profilePicPath
-                    ? `${profilePicPath}`
-                    : DEFAULT_PROFILE_PIC_PATH,
-                }}
-                resizeMode={FastImage.resizeMode.stretch}
+              style={styles.profileImage}
+              source={{
+                uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+              }}
+              resizeMode={FastImage.resizeMode.stretch}
             />
-            { __renderAddIcon() }
-            </View>
-        </TouchableNativeFeedback>
+            {renderAddIcon()}
+          </View>
+        </CircleGradient>
+      </TouchableNativeFeedback>
     </View>
-}
+  );
+};
 
 let styles = StyleSheet.create({
-    addCircle: {position: 'absolute', top: 25, left: 25},
-    profileImage: {
-        width: 100,
-        height: 100,
-        borderRadius: 100,
-        // marginBottom: 12
-    },
-    profileImageContainer: {
-        width: 100,
-        borderRadius: 100,
-    },
-    wrapImageProfile: {
-        marginTop: 14,
-        flexDirection: 'column',
-    },
-})
+  addCircle: {position: 'absolute', top: 25, left: 25},
+  profileImage: {
+    width: 97,
+    height: 97,
+    borderRadius: 100,
+    marginLeft: 1.4,
+    marginTop: 2,
+    marginBottom: 2
+  },
+  profileImageContainer: {
+    width: 100,
+    borderRadius: 100
+  },
+  wrapImageProfile: {
+    flexDirection: 'column'
+  }
+});
+
+ProfilePicture.propTypes = {
+  onImageContainerClick: PropTypes.func,
+  profilePicPath: PropTypes.string,
+  disabledAddIcon: PropTypes.bool,
+  karmaScore: PropTypes.number
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

PropTypes have been added for the new props, which is a good practice for type checking in React. However, consider adding `.isRequired` to any props that are necessary for the component to function correctly. For example, if `onImageContainerClick` is a required function, it should be marked as such to avoid potential errors.
```diff
-  onImageContainerClick: PropTypes.func,
+  onImageContainerClick: PropTypes.func.isRequired,
```

Overall, these changes introduce a new feature to display a user's karma score in a visually appealing way. However, it's important to ensure that these changes are thoroughly tested and that all edge cases are handled correctly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->